plugins/org.jboss.tools.jst.js/plugin.xml

```diff
@@ -36,20 +36,11 @@
    </extension>
    <extension
       point = "org.eclipse.ui.newWizards">
-      <category 
-            id="org.jboss.tools.jst.js.Bower" 
-            name="Bower">
-      </category>
-      <category
-            id="org.jboss.tools.jst.js.BowerInit"
-            parentCategory="org.jboss.tools.jst.js.Bower"
-            name="Bower Init">
-      </category>
       <wizard
             id="org.jboss.tools.jst.js.wizards.bower.init"
             name="Bower Init"
             class="org.jboss.tools.jst.js.bower.internal.wizard.BowerInitWizard"
-            category="org.jboss.tools.jst.js.Bower/org.jboss.tools.jst.js.BowerInit"
+            category="org.eclipse.wst.jsdt.ui.java"
```

Comment:
jst.js plugin/feature doesn't require wst.jsdt so it can be installed w/o jsdt. What will happen if there is not jsdt.ui.java category?